externallib/DPA/DP.py

```diff
@@ -0,0 +1,152 @@
+"""
+Module implementing the "Fast Search and Find of Density Peaks" clustering algorithm of Rodriguez and Laio (2014).
+"""
+
+import numpy as np
+import scipy as sp
+from scipy import stats
+from sklearn.neighbors import NearestNeighbors
+
+
+class Density_Peaks_clustering:
```

Comment:
It would be good to highlight the differences between this class and **LAIO_DB**. I've tried to document these classes myself but this has been difficult given that I didn't write the code for either.

Reply:
Would it be possible to add a little documentation so that I may understand your implementation of the algorithm?

Reply:
Do you want me to document the Laio_DB thingie in asaplib/clustering?

Reply:
Yep sorry for the delayed response on this. That would be great because it would allow me to understand e.g. what the returned **cl** variable is in the fit method of the **Laio_DB** class. This will allow me to ensure that the fit method in the new class returns an object of the same type.

Reply:
I see. Will do it in the next a couple of days.

Reply:
I documented the old LAIO_DB class